domador.js

```diff
@@ -41,6 +41,9 @@ var shallowTags = [
 var paragraphTags = [
   'ADDRESS', 'ARTICLE', 'ASIDE', 'DIV', 'FIELDSET', 'FOOTER', 'HEADER', 'NAV', 'P', 'SECTION'
 ];
+var blockTags = [
+  'ADDRESS', 'ARTICLE', 'ASIDE', 'DIV', 'FIELDSET', 'FOOTER', 'HEADER', 'NAV', 'P', 'SECTION', 'UL', 'LI', 'BLOCKQUOTE', 'BR'
```

Comment:
Nested tables? I guess there's no reason not to; I can give it a try and if it causes unexpected issues people can file them.